### ISSUE TITLE
Missing google-play-services_lib referenced in project.properties

### ISSUE THREAD

**Issue by mdivkovic** [GuJEMSAdTech/gujemsandroidsdk#1]
You have a line like this in the project.properties file:

`android.library.reference.1=../google-play-services_lib`

As far as I can see, this directory/file is nowhere to be found and while attempting to convert this into a gradle project, it fails with an unresolvable error  due to this.

**Comment by mr8ungle** [comment]
Hi,

the Google Play Services library is not bundled with the SDK but should be installed via ADT / the Android SDK manager. Instructions below include steps for Eclipse with ADT as well as Android Studio with Gradle.

https://developers.google.com/android/guides/setup

We will add a gradle build in the coming days.

Best,
G+J e|MS

**Comment by mdivkovic** [comment]
Hey,

first off, thanks for the very quick reply :)

I managed to resolve my issue by removing this line from project.properties before starting the import to Android Studio, and then manually adding the play services dependency to build.gradle, so all is well now. Looks like the Eclipse-to-Gradle conversion of Android Studio doesn't handle these cases very gracefully at this point.

We moved from Eclipse+Ant to Android Studio+Gradle a little over a year ago so I probably forgot how things work in Eclipse-land by now.

Very much looking forward to Gradle support from you guys!

Cheers
